### DISCUSSION TITLE
One instance of OctoDash, multiple OcoPrint servers

### DISCUSSION THREAD

**Discussion by RNRMentor** [UnchartedBull/OctoDash#1324] [Q&A] [answered]
Is it possible to use a single instance of OctoDash to control multiple OctoPrint servers?  It seems like when you start a print, you are locked into monitoring that single print.

**Comment by UnchartedBull** [comment, marked as answer]
not possible atm. https://github.com/UnchartedBull/OctoDash/issues/1148 is the issue for that

**Reply by RNRMentor** [reply]
Thank you for the reply. I plus oned issue #1148.  Thank you for all that you've done on this project.  I love it.
